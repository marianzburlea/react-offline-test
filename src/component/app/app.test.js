import React from 'react'
import { create } from 'react-test-renderer'
import App from '.'

describe('@App', () => {
  it('should match the snapshot', () => {
    const app = create(<App />)

    expect(app.toJSON()).toMatchSnapshot()
  })
})
