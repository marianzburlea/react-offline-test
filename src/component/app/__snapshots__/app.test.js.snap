// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@App should match the snapshot 1`] = `
<div>
  <h1>
    UK Energy Mix
  </h1>
  <div
    className="sc-gqjmRU jGsDzS"
  >
    0
  </div>
</div>
`;
